tests/integration/v3_auth_test.go

```diff
@@ -583,86 +506,3 @@ func TestV3AuthWatchErrorAndWatchId0(t *testing.T) {
 
 	<-watchEndCh
 }
-
-func TestV3AuthWithLeaseTimeToLive(t *testing.T) {
```

Comment:
It is the same verification of `authTestLeaseTimeToLive` in `ctl_v3_auth_test.go` and was added in https://github.com/etcd-io/etcd/pull/15656